mirage-server/unikernel.ml

```diff
@@ -0,0 +1,23 @@
+open Lwt.Infix
+
+module Main (R : Mirage_random.C) (M : Mirage_clock.MCLOCK) (P : Mirage_clock.PCLOCK) (T : Mirage_time_lwt.S) (S : Mirage_stack_lwt.V4) (FS: Mirage_kv_lwt.RO) = struct
+
+  module O = Openvpn_mirage.Server(R)(M)(P)(T)(S)
+
+  let read_config data =
+    FS.get data (Mirage_kv.Key.v "openvpn.config") >|= function
+    | Error e -> Rresult.R.error_to_msg ~pp_error:FS.pp_error (Error e)
+    | Ok data ->
+      let string_of_file _ = Error (`Msg "no string_of_file support") in
```

Comment:
Since this is a user-facing error, perhaps something like "The openvpn configuration contains references to external files, which this unikernel cannot read. Please inline those external files into the provided configuration." or something like that. Not blocking for this PR, just something we can consider in the future (since we do have a utility to do this inlining).